.ci/check-for-chart-changes.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
could we put the shell files, template files, etc. for CI in the `.github` folder as well?
using another directory `.ci` artificially splits the CI stuff.

Reply:
Also: a license header for all shell files would be needed (below the shebang I guess):
```
# Copyright (c) 2019 Contributors to the Eclipse Foundation
#
# See the NOTICE file(s) distributed with this work for additional
# information regarding copyright ownership.
#
# This program and the accompanying materials are made available under the
# terms of the Eclipse Public License 2.0 which is available at
# http://www.eclipse.org/legal/epl-2.0
#
# SPDX-License-Identifier: EPL-2.0
```

Reply:
done